app/views/external_works/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <!--navigation-->
 <% if logged_in_as_admin? %>
```

Comment:
This button is shown only to admins, but the link itself is accessible to all. Should we lock that down too?

Reply:
Yeah, definitely. There's probably not much chance of users finding it, but better safe than sorry.

Reply:
We forgot about this thread, let's update this before merging.